services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import QuerySet
+from db.models import MovieSession
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    if isinstance(movie_show_time, str):
+        movie_show_time = datetime.datetime.strptime(
+            movie_show_time, "%Y-%m-%d %H:%M:%S"
+        )
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        session_date = datetime.datetime.strptime(
+            session_date, "%Y-%m-%d"
+        ).date()
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> QuerySet:
+
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
Look you already have method for getting MovieSession by id ( `get_movie_session_by_id` )